inflammation/models.py

```diff
@@ -44,3 +44,13 @@ def daily_min(data):
     """
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```

Comment:
there is a numpy function that performs the standard deviation. maybe it's easier to use it instead of calculating the standard deviation manually. the function is "numpy.std"